ui/app/components/clients/page/sync.hbs

```diff
@@ -22,13 +22,15 @@
       </:subTitle>
 
       <:stats>
-        <StatText
-          @label="Average sync clients per month"
-          @value={{this.average this.byMonthActivityData "secret_syncs"}}
-          @size="m"
-          class="data-details-top has-top-padding-l"
-          data-test-average-sync-clients
-        />
+        {{#if (not-eq (this.average this.byMonthActivityData "secret_syncs") 0)}}
```

Comment:
We could prevent computing the average twice by wrapping this in
```
{{#let (this.average this.byMonthActivityData "secret_syncs") as |avg|}}
```
and then checking that against eq and passing it in as value

Reply:
I thought someone might bring this up 😅